### ISSUE TITLE
WebLogs should use localtime() instead of gmtime()

### ISSUE THREAD

**Issue by uhelp** [ProgVal/Supybot-plugins#293]
in order to display proper time i adjusted plugins/WebLogs line 178 from gmtime = time.gmtime(int(words[0]))  to gmtime = time.localtime(int(words[0]))
The server is running in TZ Germany/Berlin and hence showing alwaya "wrong" time.

Subsequently var nameing in some more lines should be adjusted to.

**Comment by ProgVal** [comment]
I know using UTC is not the best solution, but I am not sure using the server time is much better. It is still using an arbitrary timezone independent from the user.

**Comment by uhelp** [comment]
True. But it'll helps a lot of channels, as usually the non english language in a channel predefines most of users TZ too. GMT is valid just for one TZ

**Comment by uhelp** [comment]
well, i'm going to make this configurable

**Comment by ProgVal** [comment]
Yeah, I just had the same idea.

**Comment by uhelp** [comment]
so you are faster 8)

**Comment by ProgVal** [comment]
You can try something based on this to do the conversion: https://github.com/ProgVal/Limnoria/blob/master/plugins/Time/plugin.py#L193-L206

**Comment by uhelp** [comment]
OK. Will try. Nice job for a beginner like me. Might take some time.
